### PR TITLE
added the CREATE TABLE file

### PR DESCRIPTION
**Pull request by anengineer1**

### CODE REVIEW

File: database/create_table.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE messages(
+       id serial PRIMARY KEY,
+       message_datetime TIMESTAMP with time zone,
```
Comment (deavid) [inline database/create_table.sql:3]: field should be `created_at_utc` TIMESTAMP NOT NULL

File: database/create_table.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+       username VARCHAR(255),
```
Comment (deavid) [inline database/create_table.sql:4]: should be NOT NULL

File: database/create_table.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+       message TEXT
```
Comment (deavid) [inline database/create_table.sql:5]: should be NOT NULL

**Review by deavid — CHANGES_REQUESTED** [review]

File: database/create_table.sql (hunk 1 above)
Comment (anengineer1) [reply database/create_table.sql:3]: so, it litreally needs to have UTC in the varname for postgres to understand the timezone... and it doesnt need "with time zone". Correct?

File: database/create_table.sql (hunk 2 above)
Comment (anengineer1) [reply database/create_table.sql:4]: I understand, no point in sending a message with no content.

File: database/create_table.sql (hunk 1 above)
Comment (deavid) [reply database/create_table.sql]: no, it needs to be _utc for the programmer to understand that this should map to UTC, and prevent stupid stuff like parsing it as local. PostgreSQL does the same thing regardless of name.

File: database/create_table.sql (hunk 2 above)
Comment (deavid) [reply database/create_table.sql]: No content is different from NULL. NULL means it is not set.

**Review by deavid — APPROVED** [review]

**Pull request merged**